### PR TITLE
Sd prebuilt dashboards

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx [hunk 1]
```diff
@@ -28,21 +28,88 @@ Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capab
...
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+</TechTileGrid>        
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
```
Comment (nbaenam) [inline src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx:86]: That URL doesn't work for me, I get the "Oops! Something went wrong."

File: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx [hunk 2]
```diff
@@ -54,12 +121,27 @@ Browse the entire dashboard catalog to find dashboard templates for all the avai
   Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index. Browse popular pre-built dashboards you want to install using the <Icon name="fe-chevron-left"/> and <Icon name="fe-chevron-right"/> buttons, or select the **View All** link to see the entire catalog.
 </figcaption>
 
+To install a pre-built dashboard:
+
+1. Select your account in the top right account picker.
+2. From the dashboards page, search for the pre-built dashboard you want to customize.
+3. Click the pre-built dashboard to select it. A new **dashboard setup** window should appear.
+4. Follow the instructions. If you've already instrumented the data source, you can skip this step.
+5.Click **View dashboard** to deploy the dashboard and start visualizing your data. Note that you can customize your new pre-built dashboard (see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/)). You can also change the name and settings, add new content and new widgets, and refine the charts
```
Comment (nbaenam) [inline src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx:130]: Step 5 displays in the same line as step 4 in the build. The final period is missing ", and refine the charts."

File: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx [hunk 3]
```diff
@@ -28,21 +28,88 @@ Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capab
...
+3. New recommendations will now appear on your dashboards homepage.
+</Step>
 
-Browse the entire dashboard catalog to find dashboard templates for all the available integrations. Click the **View All** link next to the popular pre-built dashboards title to see and search for all the available pre-built dashboards. You can use the <Icon name="fe-chevron-left"/> and <Icon name="fe-chevron-right"/> buttons to browse more pre-built dashboards.
+
+<Step>
+### Install pre-built dashboards [#use]
 
 <img
   title="Search for a pre-built dashboard"
```
Comment (nbaenam) [inline src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx:115]: I think that this screenshot should go inside step 4.

File: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx [hunk 4]
```diff
@@ -54,12 +121,27 @@ Browse the entire dashboard catalog to find dashboard templates for all the avai
...
+4. Follow the instructions. If you've already instrumented the data source, you can skip this step.
+5.Click **View dashboard** to deploy the dashboard and start visualizing your data. Note that you can customize your new pre-built dashboard (see [Manage your dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/)). You can also change the name and settings, add new content and new widgets, and refine the charts
+</Step>
+
+<Step>
+
+### View more templates [#more-templates]
+
+Browse the entire dashboard catalog to find out-of-the-box dashboard templates for all available integrations. Click the **View All** link next to the **Recommended dashboards** title to see and search for all the available pre-built dashboards. You can use the <Icon name="fe-chevron-left"/> and <Icon name="fe-chevron-right"/> buttons to browse more pre-built dashboards.
+
```
Comment (nbaenam) [inline src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx:138]: This is where I'd add the previous screenshot.

**Review by nbaenam — DISMISSED** [review]
I just made some small suggestions! Feel free to take them or leave them🙂

File: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx:115]: Sounds good! Moving it

File: src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx [hunk 5]
```diff
@@ -28,46 +28,134 @@ Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capab
   Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index.
 </figcaption>
 
+## Dashboards tailored for your stack 
 
-## How to install pre-built dashboards [#use]
+To make full use of every tool New Relic has to offer you can install pre-built dashboards that fit your business needs. If you run an e-commerce site and you want to understand any potential revenue risk, you can install the dashboard `Customer experience bottom of the funnel analysis` which automatically monitors orders, checkout, latency, and tracks promotions. Or if you're a DevOps engineer and you want to make sure your hosts and servers don't get overwhelmed, you can install our `Infrastructure Dashboard` to quickly analyze CPU and memory and make changes.
```
Comment (akristen) [inline src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx:33]:
```suggestion
To make full use of every tool New Relic has to offer you can install pre-built dashboards that fit your business needs. If you run an ecommerce site and you want to understand any potential revenue risk, you can install the dashboard `Customer experience bottom of the funnel analysis` which automatically monitors orders, checkout, latency, and tracks promotions. Or if you're a DevOps engineer and you want to make sure your hosts and servers don't get overwhelmed, you can install our `Infrastructure Dashboard` to quickly analyze CPU and memory and make changes. 
```

**Review by akristen — APPROVED** [review]
Updated e-commerce to ecommerce, otherwise looks good! Images work etc. Really like how the last screenshot was made too!

**Pull request merged**